### PR TITLE
Add benchmark for a stopper.RunTask

### PR DESCRIPTION
**Pull request by dt**
Just wanted to know how much overhead it adds

[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/4011)

### CODE REVIEW

File: util/caller/resolver_test.go [hunk 1]
```diff
@@ -58,3 +59,13 @@ func TestDefaultCallResolver(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkSimpleCaller(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		file, line, _ := Lookup(1)
+		s := fmt.Sprintf("%s:%d", file, line)
```
Comment (tamird) [inline util/caller/resolver_test.go]: are you intentionally measuring this?

**Comment by nvanbenschoten** [comment]
What are the results? How much overhead does it add?

**Comment by tamird** [comment]
They're in the commit messages.

File: util/caller/resolver_test.go (hunk 1 above)
Comment (petermattis) [inline util/caller/resolver_test.go]: Probably worried about compiler optimization. I think you can move this inside the `testing.Verbose()` block.

**Comment by petermattis** [comment]
LGTM

File: util/caller/resolver_test.go (hunk 1 above)
Comment (dt) [inline util/caller/resolver_test.go]: Yeah, I was, but also trying to replicate what stopper does -- adding one inside, one outside makes it clearer where the overhead comes from.

**Comment by dt** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[util/caller/resolver_test.go, line 66 \[r2\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4011#-K986wQL1TOWlbZ5pTIK-r2-66:-K987hDXpSkq1W1NNpTy:1610000531) ([raw file](https://github.com/cockroachdb/cockroach/blob/a359bf6a7f99835b07c548f80eca54305f837993/util/caller/resolver_test.go#L66)):*
Added.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4011)*

**Comment by tamird** [comment]
Reviewed 1 of 1 files at r1, 1 of 1 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4011)*

**Pull request merged**